Tools/AP_Periph/can.cpp

```diff
@@ -1403,6 +1467,10 @@ void AP_Periph_FW::hwesc_telem_update()
 
 void AP_Periph_FW::can_update()
 {
+    for (uint8_t i = 0; i < HAL_NUM_CAN_IFACES; i++) {
+        can_do_dna(i);
```

Comment:
do we care about the result here?

Reply:
No, we just attempt at allocation, if it fails, we still move on. If we are here that mean at least one iface got node allocated.